### ISSUE TITLE
How to use :lang selector in atomizer?

### ISSUE THREAD

**Issue by jitendravyas** [acss-io/atomizer#344]
https://css-tricks.com/almanac/selectors/l/lang/

**Comment by sarbbottam** [comment]
@jitendravyas I would probably use classical CSS for this use case.

https://acss.io/frequently-asked-questions.html

>### Should I "atomize" everything? Should I style everything using atomic classes?
>If changing some styling requires you to edit multiple files, then you should use the classic CSS approach: apply a "meaningful" class to the element you want to style and use that hook to write rules in a style sheet. But if changing a given style across the board can be done in one place - other than a style sheet - then you should go with Atomic CSS.

>An example of the former could be headings (unless they are "components") meant to look the same across many modules, or any other styling meant to be shared across different modules. An example of the latter is a component that lives in a template or a JS file, like a specific widget with its own styling.

**Comment by jitendravyas** [comment]
Yes but I think Atomizer should support for all possible CSS pseudo-classes.

What do you say @src-code?
